src/routes/safe/components/Layout.jsx

```diff
@@ -92,6 +91,15 @@ const Layout = (props: Props) => {
   const { address, ethBalance, name } = safe
   const etherScanLink = getEtherScanLink('address', address)
 
+  const onNotificationClosesCb = () => {
+    history.push(`${match.url}/transactions`)
+  }
+
+  useEffect(() => {
```

Comment:
Let's move this to `routes/safe/container` and leave layout file only for Layout purposes :)

Reply:
Also if you implement the changes I wrote below there are no need in enhancing `fetchTransactions` action with userAddress, etc

Reply:
Done